### PR TITLE
Fix FIPS test failures due to script_output update

### PR DESCRIPTION
**Pull request by QingmingSu**
Resolve the issue that some FIPS test cases are failed after
script_output function in testapi.pm been updated with:
  /bin/bash -eox pipefail $run_script

### CODE REVIEW

File: tests/fips/fips_setup.pm [hunk 1]
```diff
@@ -21,17 +21,22 @@ use testapi;
 use strict;
 
 # Install fips pattern and update grub.cfg to boot with fips=1
-sub run() {
+sub run {
     select_console 'root-console';
 
-    if (!script_output "grep 'fips=1' /proc/cmdline | tee") {
-        script_run 'zypper -n install --type pattern fips', 300;
```
Comment (mitiao) [inline tests/fips/fips_setup.pm]: how about this? :)
if (script_output "! grep 'fips=1' /proc/cmdline") {
    zypper_call "install --type pattern fips";

Comment (QingmingSu) [inline tests/fips/fips_setup.pm]: Seems it won't work. If fips is not enabled on target, then:
script_output "! grep 'fips=1' /proc/cmdline" will return empty string.
Then afterward commands won't be run, which is actually required to install fips pattern.

Comment (mitiao) [inline tests/fips/fips_setup.pm]: ah, sorry, that's my fault.
what i tried to do is to use exit code to detect 'fips=1' is enabled or not, may like this:
my $ret = script_run "grep 'fips=1' /proc/cmdline";
if ($ret ne 0)

**Comment by okurz** [comment]
@asmorodskyi

**Comment by okurz** [comment]
Please provide a link or screenshot to a local verification run.

Call `tools/tidy` to fix the syntax.

**Comment by QingmingSu** [comment]
Updated to fix the syntax error.

Test scripts had been verified with local openQA runs:
For fips_setup.pm:
    http://147.2.212.224/tests/35

For openssl_fips_alglist.pm openssl_fips_hash.pm  openssl_fips_cipher.pm openssh_fips.pm:
    http://147.2.212.224/tests/44

For aide_check.pm and curl_fips_rc4_seed.pm:
    http://147.2.212.224/tests/45

**Pull request merged**